app/models/task_management.rb

```diff
@@ -35,7 +35,7 @@ def self.all_notifications_for(user_type, id)
   def self.update_all_notifications_as_seen(user)
     query = user.user_type == "tasker" ? "tasker_id" : "taskee_id"
     attribute = query.gsub("id", "notified")
-    where(query => user.id).where(attribute => false).
+    where(query => user.id).where(attribute => false).where(paid: true).
```

Comment:
I am suggesting we have a scope for paid tasks instead of repeating the same query multiple times around the code base. You might want to take a look the following snippets 

>class Article < ApplicationRecord
  scope :published,               -> { where(published: true) }
  scope :published_and_commented, -> { published.where("comments_count > 0") }
end